webapp/app/templates/components.html

```diff
@@ -152,12 +152,12 @@ <h5 class="mb-0">{{ card['heading'] }}</h5>
                 <div class="row collapsed" data-toggle="collapse"
                             data-target="#details-body-{{ details_id }}"
                             aria-expanded="false" aria-controls="collapseOne">
-                    <button class="btn btn-primary container" type="button">
+                    <button class="container" type="button">
                         <div class="row">
                             <div class="details-icon col-1 control-show-more"></div>
                             <div class="details-icon col-1 control-show-less"></div>
                             <div class="col">
-                                    <h5 class="mb-0">{{title}}</h5>
+                                <span class="mb-0">{{title}}</span>
```

Comment:
You exchanged it with span because otherwise we jump from h1 to h5, correct?

Reply:
Exactly.